### ISSUE TITLE
word2vec

### ISSUE THREAD

**Issue by mudabek** [ivonajdenkoska/variational-xray-report-gen#1]
Hello! Thank you for sharing your work! Running the training requires bio_word2vec_weights.pkl file. Can you please tell where I can get this file?

**Comment by ivonajdenkoska** [comment]
You can download the file from here [https://github.com/ncbi-nlp/BioSentVec](https://github.com/ncbi-nlp/BioSentVec) ([BioWordVec vector] https://ftp.ncbi.nlm.nih.gov/pub/lu/Suppl/BioSentVec/BioWordVec_PubMed_MIMICIII_d200.vec.bin).

**Comment by mudabek** [comment]
ok thank you very much! I downloaded some file from gensim related to word2vec, but then it weighed only 1GB. and when I use it for training I am getting stuck at the gradient backpropagation.

Another issue I faced was when calculating priors I was getting negative values of the variance from logit outputs. But then I added sigmoid at the output of priors. But I guess all issues probably started because of me using the wrong word2vec.

Also, the one you use in the code is .pkl file. Is it just pickled version of the file you shared?

**Comment by mudabek** [comment]
model.vectors.shape
Out[20]: (16545452, 200)

That's what I am getting when I am reading the file using gensim's keyedvectors. Does that look right? So I just pass this vector as embedding initialization?

**Comment by mudabek** [comment]
I tested it with these embeddings, and it still falls because of negative values of prior's variance

**Comment by hlk-1135** [comment]
> Hello! Thank you for sharing your work! Running the training requires bio_word2vec_weights.pkl file. Can you please tell where I can get this file?

Hi,did you find the bio_word2vec_weights.pkl file?

**Comment by NorthTree22** [comment]
Hi, have you solved the problem yet?

**Comment by yuxue-liu** [comment]
> > Hello! Thank you for sharing your work! Running the training requires bio_word2vec_weights.pkl file. Can you please tell where I can get this file?
> 
> Hi,did you find the bio_word2vec_weights.pkl file?

Hi, have you solved the problem yet?

**Comment by ivonajdenkoska** [comment]
Hi. I assume the format of the file for these pre-trained weights provided here https://github.com/ncbi-nlp/BioSentVec, was changed meanwhile. Maybe you can try using the currently provided format and load them like that, possibly using fasttext library.
